packages/nimble-components/src/rich-text-editor/types.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
Reminder to add a `types.spec.ts` test file that follows the pattern of other type tests.